README.md

```diff
@@ -36,11 +36,13 @@ Adding startup logic, [basic](https://github.com/behance/docker-base#startuprunt
 - [S6](https://github.com/just-containers/s6-overlay): PID 1 zombie reaping, startup coordination, shutdown signal transferal. Nginx and PHP are preconfigured to shutdown as gracefully as possible.
 - [Goss](https://goss.rocks): for serverspec-like testing. Run `goss -g /tests/php-fpm/{PHP_MAJOR.PHP_MINOR}(-variant).goss.yaml` to validate any configuration updates
 - Ubuntu (default) or Alpine OS [base](https://github.com/behance/docker-base)
-- Extra PHP Modules:
+- Common PHP extensions:
+
+For extension customization, including enabling and disabling defaults, see [here](https://github.com/bryanlatten/docker-php#downstream-configuration)
 
 `*`  - not available on `-alpine` variant  
-`^`  - not available on `7.2` 
-`~`  - disabled by default (default: use `phpenmod` to enable, Alpine-only: uncomment .ini file)
+`^`  - not available on `7.2`  
+`~`  - disabled by default  
```

Comment:
Those markdown newlines are invaluable.

Reply:
actually, the are *very* valuable. the linebreak doesn't take place unless there are two of them, there is currently one.

Reply:
I think that's what I just said? I actually had to google it cuz I doubted myself, ha.

![image](https://user-images.githubusercontent.com/250121/31836531-ace66fb0-b5a3-11e7-8d8a-393be0d8c04f.png)

Reply:
haha, my apologies. I read it too fast.